README.md

```diff
@@ -1,4 +1,8 @@
-# [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) LORIS Neuroimaging Platform 
+[![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) 
+[![Minimum PHP Version](https://img.shields.io/travis/php-v/aces/loris/22.0-release?color=787CB5)](https://php.net/)
```

Comment:
should this point to branch master just like status? If pointing to 22.0, it might have to be updated for each (major) release of LORIS.